redislock_test.go

```diff
@@ -1,273 +1,197 @@
 package redislock_test
 
 import (
-	"context"
-	"errors"
 	"math/rand"
 	"sync"
 	"sync/atomic"
 	"testing"
 	"time"
 
-	. "github.com/bsm/redislock"
-	"github.com/go-redis/redis/v8"
+	"github.com/joshtechnologygroup/redislock"
+	"github.com/mediocregopher/radix/v3"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
 )
 
-const lockKey = "__bsm_redislock_unit_test__"
+const lockKey = "__joshtechnology_redislock_unit_test__"
 
-var redisOpts = &redis.Options{
-	Network: "tcp",
-	Addr:    "127.0.0.1:6379", DB: 9,
-}
-
-func TestClient(t *testing.T) {
-	ctx := context.Background()
-	rc := redis.NewClient(redisOpts)
-	defer teardown(t, rc)
-
-	// init client
-	client := New(rc)
-
-	// obtain
-	lock, err := client.Obtain(ctx, lockKey, time.Hour, nil)
-	if err != nil {
-		t.Fatal(err)
-	}
-	defer lock.Release(ctx)
-
-	if exp, got := 22, len(lock.Token()); exp != got {
-		t.Fatalf("expected %v, got %v", exp, got)
-	}
-
-	// check TTL
-	assertTTL(t, lock, time.Hour)
-
-	// try to obtain again
-	_, err = client.Obtain(ctx, lockKey, time.Hour, nil)
-	if exp, got := ErrNotObtained, err; !errors.Is(got, exp) {
-		t.Fatalf("expected %v, got %v", exp, got)
-	}
-
-	// manually unlock
-	if err := lock.Release(ctx); err != nil {
-		t.Fatal(err)
-	}
-
-	// lock again
-	lock, err = client.Obtain(ctx, lockKey, time.Hour, nil)
-	if err != nil {
-		t.Fatal(err)
-	}
-	defer lock.Release(ctx)
-}
+var _ = Describe("Client", func() {
```

Comment:
```suggestion
_ := Describe("Client", func() {
```

Reply:
Such declaration is not allowed. Getting `expected declaration, found _`